integration_tests/seeds/materialization/materialization_seed.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+seeds:
+  - name: tst_fct_chained_views
```

Comment:
do you mind updating the seed name to have the `test_` prefix for consistency?

Reply:
Done!